rwengine/src/engine/SaveGame.cpp

```diff
@@ -1362,7 +1364,9 @@ std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     rwfs::path gamePath(homedir);
     gamePath /= gameDir;
 
-    if (!rwfs::exists(gamePath) || !rwfs::is_directory(gamePath)) return {};
+    if (!rwfs::exists(gamePath) || !rwfs::is_directory(gamePath)) {
+        return {};
```

Comment:
nit: maybe return `std::optional`.

Reply:
I bumped onto this https://github.com/HiroakiMikami/deep-coder/issues/25